### ISSUE TITLE
[Wallet Redesign] Summary

### ISSUE THREAD

**Issue by gibbsfromncis** [bitshares/bitshares-ui#2668]
This is to collect all issues related to redesign of our current wallet. 

**Features**:
1. Layout - https://github.com/bitshares/bitshares-ui/issues/2667
2. Landing Page
3. Dashboard
4. Manage Funds
5. Wallet
6. Beginners Guide
7. Orders
8. Business
9. Blockchain
10. Governance
11. Settings
12. Language Select / Propose
13. News
14. Invite & Earn
15. Membership
16. Permissions
17. Contacts
18. Accounts

## Layout

**Comment by froooze** [comment]
How does this apply to Ross UI?

**Comment by gibbsfromncis** [comment]
@froooze totally nohow. Beginning from the new worker proposal we're working on complete redesign of bitshares-ui interface. We started discussions in summer of 2018. And now, beginning from new worker proposal, I'm a person responsible for User Interface and User Experience (UI/UX) of our wallet and head of the design team. 

Beginning from my first day I promote the idea of new modern interface which will make all our features stack more user-friendly. Another thing is me, as a technical guy with a great development and UX background, see that complete redesign will also have to restructure our code-base and decrease entering barrier for new freelancers. This will help us to speed-up development process, release new features and kill bugs more faster. My personal(!) mind is that more developers attraction may even decrease hourly rate we pay for our freelancers. So our benefits will be - more work, faster for less expenses. (But let me notice it is my personal mind and I'm not eligible to make any statements like that because my zone of responsibility is UI/UX. I'm just talking about potential benefits we may get) 

We have the following plan:
- Create mockups of most important parts of current Wallet. 
- Create design based on mockups
- Implement new design as MVP (Minimal Viable Product) (we will host it separate and it will be accessible by different link)
- Get first feedback and continue redesign of other bitshares-ui features and their integration on new wallet. 

@froooze, if you're ready to pay attention and participate on redesign we can create a telegram chat where I will send updates and you can share your feedback. Also, we can attract more users in this chat. I have plans to create it but after release of MVP (but why we should wait? Lets do it know if you wish to help :)

**Comment by froooze** [comment]
@gibbsfromncis: Yes, I wanna help and join the redesign process.

**Comment by gibbsfromncis** [comment]
@froooze please add me in telegram - @gibbsfromncis

**Comment by gibbsfromncis** [comment]
Latest updates:
**Design:**
The Final version of a Dashboard (#2850)

**Comment by sschiessl-bcp** [comment]
Please create issues that match the bounty assigned and work done as mentioned in the invoice, put it in the correct milestone and ping me in the new issue.

**Comment by gibbsfromncis** [comment]
@sschiessl-bcp 
#2936 190607
#2937 190621

**Comment by gibbsfromncis** [comment]
Update:

#2986 - The first draft of Landing Page without texts (blocked by https://github.com/bitshares/bitshares-ui/issues/2986)

https://github.com/bitshares/bitshares-ui/issues/2984 - The first version of Beginners Guide without all screens (blocked by https://github.com/bitshares/bitshares-ui/issues/2985)

https://github.com/bitshares/bitshares-ui/issues/2987 - Mobile Dashboard and Mobile Menu (Layout)

**Comment by gibbsfromncis** [comment]
190705 https://github.com/bitshares/bitshares-ui/issues/3032
